modules/tomcat/templates/tomcat.conf.epp

```diff
@@ -36,7 +36,7 @@ CATALINA_TMPDIR="/var/cache/tomcat/temp"
 
 # You can pass some parameters to java here if you wish to
 #
-JAVA_OPTS="-Xmin<%= $java_ram %> -Xmax<%= $java_ram %>"
+JAVA_OPTS="-Xmin<%= $java_heap %> -Xmax<%= $java_heap %>"
```

Comment:
Are you sure  that java heap memory setting are called Xmin and Xmax but not Xms and Xmx?